### DISCUSSION TITLE
No ET reading anymore

### DISCUSSION THREAD

**Discussion by DanRobCC** [artisan-roaster-scope/artisan#1202] [General]
Artisan has stopped showing and recording ET on both our machines, Doesn't show up throughout the roast and is not recorded on the profiles afterwards, nothing reading on the side panels, just 0,0 using 30kg Toper connected via ethernet via MODBUS and a 10kg Toper using a phidget.

At first i thought it may be an issue with the temp probes but its happening on both machines and I cannot figure out why.

Any advice?

**Comment by poundy** [comment]
Settings most likely. Send your settings file here. Save the settings out, rename the file with a .txt extension, and drag and drop here on the website (not via email). Send us a sample roast log as well.

**Reply by DanRobCC** [reply]
Not really sure now to send over my settings. Never done that before.

**Reply by poundy** [reply]
You can export your Artisan settings via menu Help >> Export settings into a file artisan-settings.aset. Rename this file so it has a .txt extension and then drag and drop this file to a comment box here on this discussion on GitHub.

**Reply by DanRobCC** [reply]
Hope these are correct and you can view them, Cheers!

[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/11785891/artisan-settings.txt)
![RZA_ROAST_23-06-15_1521 alog](https://github.com/artisan-roaster-scope/artisan/assets/136587611/c997dcda-d71a-4fb0-8932-481d78f3d8d7)

**Reply by MAKOMO** [reply]
You are using a recent continuous build of Artisan (v2.8.3).

Those settings show a MODBUS setup for the main device (ET/BT), most likely using the standard Toper PLC machine setup. There is one extra device of type Phidgets TMP1200 configured for 4-wire PT1000 . The data of that device (with LCD/curve not visible) is routed to the ET and overwriting its MODBUS data. That ET channel should be visible, but most likely does not receive any data.

First thing to do is to check your Phidget setup. Do you have the latest Phidgets driver from phidgets.com installed on this machine? Does the Phidget Control Panel app that is installed with that driver shows the TMP1200 properly connected and on click shows valid temperature readings?

**Reply by DanRobCC** [reply]
I have the latest driver installed and both channels 0 and 1 on the panel are showing temperature readings.

**Reply by MAKOMO** [reply]
The panel of the roasting machine? This is not relevant here as your configuration is overwriting that one with the data received from the Phidget TMP1200 module. What is shown in the Phidget Control Panel? Can you make upload screenshot of it?

**Reply by DanRobCC** [reply]
I was talking about the phidget control panel sorry

**Reply by MAKOMO** [reply]
Both channels? The settings are only talking about one TMP1200 module which makes up only one channel. I am more and more confused by your reports. Does Artisan reports to connect to the Phidgets module in its message line on ON (or in the message window accessible from the Help menu)?

Could you please send us your Artisan log files? For this just hold the ALT key and press the plus icon in the upper left corner of the Artisan main window. This should open an email with the log files attached. Please forward this message to `info@artisan.plus` and I will take a look.

**Reply by MAKOMO** [reply]
Thanks for sending the log files. The Phidget device log tells me that you do not have a TMP1200 Phidget module connected as your Artisan settings file assumes, but a TMP1101.

You need to set your extra device (menu Config >> Devices, 2nd tab) from the current "Phidget TMP1200 1xRTD A" to the correct "Phidget TMP1101 4xTC01".

Then the first TC connected to your TMP1101 (chan0) will get mapped to the ET according to your configuration.

**Comment by DanRobCC** [comment]
That worked perfect, Thank you so much for your help!

On Mon, 19 Jun 2023 at 14:33, Marko Luther ***@***.***> wrote:

> Thanks for sending the log files. The Phidget device log tells me that you
> do not have a TMP1200 Phidget module connected as your Artisan settings
> file assumes, but a TMP1101.
>
> You need to set your extra device (menu Config >> Devices, 2nd tab) from
> the current "Phidget TMP1200 1xRTD A" to the correct "Phidget TMP1101
> 4xTC01".
>
> Then the first TC connected to your TMP1101 (chan0) will get mapped to the
> ET according to your configuration.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/artisan-roaster-scope/artisan/discussions/1202#discussioncomment-6219537>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BASCSW2OOEZBQKI7CCIWFITXMBIKTANCNFSM6AAAAAAZGGQHN4>
> .
> You are receiving this because you authored the thread.Message ID:
> <artisan-roaster-scope/artisan/repo-discussions/1202/comments/6219537@
> github.com>
>


-- 
Kind regards,

Dan Roberts

Production Roaster

www.crosbycoffee.co.uk

@crosbycoffeeltd

0151 538 5454 <01515385454>

<https://www.instagram.com/crosbycoffeeltd/>
<https://www.facebook.com/crosbycoffeeltd>
<https://twitter.com/crosbycoffeeltd>

**Comment by DanRobCC** [comment]
Only problem now is that the artisan ET is reading as 25 degrees when its
230 on the toper console.

On Mon, 19 Jun 2023 at 15:10, Dan Roberts ***@***.***>
wrote:

> That worked perfect, Thank you so much for your help!
>
> On Mon, 19 Jun 2023 at 14:33, Marko Luther ***@***.***>
> wrote:
>
>> Thanks for sending the log files. The Phidget device log tells me that
>> you do not have a TMP1200 Phidget module connected as your Artisan settings
>> file assumes, but a TMP1101.
>>
>> You need to set your extra device (menu Config >> Devices, 2nd tab) from
>> the current "Phidget TMP1200 1xRTD A" to the correct "Phidget TMP1101
>> 4xTC01".
>>
>> Then the first TC connected to your TMP1101 (chan0) will get mapped to
>> the ET according to your configuration.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/artisan-roaster-scope/artisan/discussions/1202#discussioncomment-6219537>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BASCSW2OOEZBQKI7CCIWFITXMBIKTANCNFSM6AAAAAAZGGQHN4>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <artisan-roaster-scope/artisan/repo-discussions/1202/comments/6219537@
>> github.com>
>>
>
>
> --
> Kind regards,
>
> Dan Roberts
>
> Production Roaster
>
> www.crosbycoffee.co.uk
>
> @crosbycoffeeltd
>
> 0151 538 5454 <01515385454>
>
> <https://www.instagram.com/crosbycoffeeltd/>
> <https://www.facebook.com/crosbycoffeeltd>
> <https://twitter.com/crosbycoffeeltd>
>
>
>

-- 
Kind regards,

Dan Roberts

Production Roaster

www.crosbycoffee.co.uk

@crosbycoffeeltd

0151 538 5454 <01515385454>

<https://www.instagram.com/crosbycoffeeltd/>
<https://www.facebook.com/crosbycoffeeltd>
<https://twitter.com/crosbycoffeeltd>

**Comment by DanRobCC** [comment]
The 10kg toper we have is using the phidget, I unplugged the usb and only left the ethernet in for the 30kg, there is still no ET reading after making this change.
